templates/detail.html

```diff
@@ -10,13 +10,46 @@
         <!-- jQuery is already downloaded and installed for you -->
         <!-- In order to use jQuery in your JavaScript, this file needs to be added BEFORE your custom JavaScript -->
         <script type="text/javascript" src="/static/js/jquery.min.js"></script>
+
+
     </head>
     <body>
         <!-- Your HTML should be here -->
+        <a href="/"> <h3>Return to main page</h3></a>
+        <div class="container">
+            <div class="name"></div>
+            <div class="img"></div>
+            <h3>Types</h3>
+            <ul>
+                <div class="types"></div>
+            </ul>
+            <h3>Description </h3>        
+            <div class="description"></div>
+            
+            <h3>Evolutions </h3>
+            <table>
+                <tr> 
+                    <th>Pokemon   </th>
+                    <th>Level   </th>
+                    <th>Method   </th>
+                </tr>
```

Comment:
I think that evolutions can be a `tbody` tag here and then you can append your rows and data to the tbody.

`<tbody class="js-evolutions"></tbody>`